README.md

```diff
@@ -3,406 +3,232 @@
+**Easy to include in a continuous integration system**<br/>
+You can run TestCafe from a console, and its reports can be viewed in a CI system's interface
+(TeamCity, Jenkins, Travis & etc.)
 
 ## Getting Started
 
-### Installing TestCafe
+### Installation
 
-Ensure that [Node.js](https://nodejs.org/) and [npm](https://www.npmjs.com/) are installed on your computer, then run a single command:
+Ensure that [Node.js](https://nodejs.org/) (version 4 or newer) and [npm](https://www.npmjs.com/) are installed on your computer before running it:
 
-```bash
+```sh
 npm install -g testcafe
 ```
 
-For more information, see [Installing TestCafe](https://devexpress.github.io/testcafe/documentation/using-testcafe/installing-testcafe.html).
-
-### Creating a Test
-
-TestCafe allows you to write tests using TypeScript or JavaScript (with its modern features like `async/await`).
-By using TypeScript to write your TestCafe tests, you get the advantages of strongly-typed languages such as: rich coding assistance, painless scalability, check-as-you-type code verification, and much more. For more information about writing tests in TypeScript, see [TypeScript Support](https://devexpress.github.io/testcafe/documentation/test-api/typescript-support.html).
-
-To create a test, create a new .js or .ts file anywhere on your computer.
-This file must have a special structure: tests must be organized into fixtures.
-
-Firstly, import the `testcafe` module.
+### Creating the Test
 
-```js
-import { Selector } from 'testcafe';
-```
+As an example, we are going to test the [https://devexpress.github.io/testcafe/example](https://devexpress.github.io/testcafe/example) page.
 
-Then declare a fixture using the [fixture](https://devexpress.github.io/testcafe/documentation/test-api/test-code-structure.html#fixtures) function.
+Create a `.js` or `.ts` file on your computer.
+Note that it needs to have a specific structure: tests must be organized into fixtures.
+You can paste the following code to see the test in action:
 
 ```js
-fixture `Getting Started`
-```
-
-In this tutorial, you will create a test for the [https://devexpress.github.io/testcafe/example](https://devexpress.github.io/testcafe/example) sample page.
-Specify this page as a start page for the fixture by using the [page](https://devexpress.github.io/testcafe/documentation/test-api/test-code-structure.html#specifying-the-start-webpage) function.
+import { Selector } from 'testcafe'; // first import testcafe selectors
 
-```js
-fixture `Getting Started`
-    .page `https://devexpress.github.io/testcafe/example`;
-```
+fixture `Getting Started`// declare the fixture
+    .page `https://devexpress.github.io/testcafe/example`;  // specify the start page
 
-Then, create the [test](https://devexpress.github.io/testcafe/documentation/test-api/test-code-structure.html#tests) function where you will place test code.
 
-```js
-import { Selector } from 'testcafe';
+//then create the test function where you'll place the code:
```

Comment:
Let's write just `then create a test and place your code there`? Or smth like this.
 I think it's better because without redundant information and this sentence is easier